doc/user/bmp.rst

```diff
@@ -57,6 +57,51 @@ The `BMP` implementation in FRR has the following properties:
 
 - monitoring peers with :rfc:`5549` extended next-hops has not been tested.
 
+RFC 7854
+========
+Unsupported features (non exhaustive):
+  - Per-Peer Header
+    
+    - Peer Type Flag
+    - Peer Distingsher
+  
```

Comment:
s/Distingsher/Distinguisher
Is not RD supported as per commit https://github.com/FRRouting/frr/pull/11800/commits/38cd75d7384110edcbc35882140914d5d4794a53#diff-af2d89aaf8862c57f4f02385cb1c5850844025ed3710d9be119a8b98af85d0ceL982 and below ?

Reply:
no, peer type & distinguisher / rd is supported for RFC9069 (BMP LocRib) but not RFC7854 (BMP).
this means the peer type & RD / peer distinguisher will always be 0 non-LocRib messages as it already was before